src/registrar/templates/admin/change_form_no_submit.html

```diff
@@ -0,0 +1,20 @@
+{% extends "admin/change_form.html" %}
+
+{% comment %} Replace the Django ul markup with a div. We'll edit the child markup accordingly in change_form_object_tools {% endcomment %}
+{% block object-tools %}
+{% if change and not is_popup %}
```

Comment:
Do we want to display anything here when if not change or is_popup? I don't think we do from a past conversation (grabbed from a template?) but if I am misremembering then that may need to be handled.

Reply:
I wouldn't mess with it (at least not in this PR). This is how the [original template](https://github.com/django/django/blob/a7c73b944f51d6c92ec876fd7e0a171e7c01657d/django/contrib/admin/templates/admin/change_form.html#L27) is written in django.

Reply:
Yeah lets conform with the template for now